src/pages/index.js

```diff
@@ -165,12 +271,21 @@ previewModalCloseButton.addEventListener("click", () => {
   closeModal(previewModal);
 });
 
-editFormElement.addEventListener("submit", handleEditFormSubmit);
-cardFormElement.addEventListener("submit", handleAddCardSubmit);
+avatarSubmitButton.addEventListener("click", () => {
```

Comment:
`avatarSubmitButton`


You don't need to close the modal when you click on the `submit button`.  

You need to close modals only in `then` blocks after a **successful response** from the server so that when there is an error, it would be opened (the inputs will not be cleared and the user could send the data again to the server), and if you change the button text, you could see the change as well. **Please, correct it everywhere in the code**